server/src/main/java/org/elasticsearch/common/util/set/Sets.java

```diff
@@ -131,7 +141,7 @@ public static <T> Set<T> union(Set<T> left, Set<T> right) {
      *
      * @param set1 the first set
      * @param set2 the second set
-     * @return the unmodifiable intersection of the two sets
+     * @return unmodifiable intersection of the two sets
```

Comment:
I'd let the native English speakers have the final say here, but it seems to me that the article here is needed - I would left the comment as is.

Reply:
I think "the intersection" would mean some unique instance, like a singleton. But sure, I'll revert it.

Reply:
Check also the `toUnmodifiableSortedSet()` javadoc:
https://github.com/elastic/elasticsearch/pull/105350/files#diff-7c90eb30306ff988dc534d8f46ab45aa269ae2037260c3cda8048e87e63bf7dcR123